internal/goeland/filters/filter.go

```diff
@@ -154,8 +155,14 @@ func filterDigestGeneric(source *goeland.Source, level int, useFirstEntryTitle b
 	if useFirstEntryTitle && len(source.Entries) > 0 {
 		digest.Title = source.Entries[0].Title
 	}
+	previousSourceTitle := ""

```

Comment:
Probably better for it to be nil instead of ""

Reply:
Renamed the variable and changed it to point to the previous entry.Source see 15ce0f7